routes/index.js

```diff
@@ -279,4 +279,121 @@ router.get('/archive', function (req, res) {
   });
 });
 
+/**
+ * POST /render
+ */
```

Comment:
This comment was brought to you by the Department of Redundancy Department

Reply:
I just followed the convention that was already in this repo. I'll remove it but other routes need to be updated as well